src/test/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStoreTest.kt

```diff
@@ -42,7 +46,37 @@ class ParticipantProjectSpeciesStoreTest : DatabaseTest(), RunsAsUser {
   }
 
   @Nested
-  inner class FetchLastUpdatedSpeciesTime {
+  inner class FetchLastCreatedSpeciesTime {
+    @Test
+    fun `fetches the last created species time for a given project`() {
+      val participantId = insertParticipant()
+      val projectId = insertProject(participantId = participantId)
+      val speciesId1 = insertSpecies()
+      val speciesId2 = insertSpecies()
+      val speciesId3 = insertSpecies()
+
+      insertParticipantProjectSpecies(
+          createdTime = Instant.EPOCH, projectId = projectId, speciesId = speciesId1)
+      insertParticipantProjectSpecies(
+          createdTime = Instant.EPOCH.plusSeconds(2), projectId = projectId, speciesId = speciesId2)
```

Comment:
Maybe set `modifiedTime` to a different value to make sure the function is actually returning the created time?